api.md

```diff
@@ -28,19 +28,35 @@ can also be done through our `REST API`
 - [3. `JSON` serializing](#3-json-serializing)
 - [4. Listing `timers` and `items` and validating updates](#4-listing-timers-and-items-and-validating-updates)
 - [5. Error handling in `ErrorView`](#5-error-handling-in-errorview)
-- [5.1 Fixing tests](#51-fixing-tests)
+  - [5.1 Fixing tests](#51-fixing-tests)
 - [6. Basic `API` Testing Using `cUrl`](#6-basic-api-testing-using-curl)
   - [6.1 _Create_ an `item` via `API` Request](#61-create-an-item-via-api-request)
   - [6.2 _Read_ the `item` via `API`](#62-read-the-item-via-api)
   - [6.3 Create a `Timer` for your `item`](#63-create-a-timer-for-your-item)
   - [6.4 _Stop_ the `Timer`](#64-stop-the-timer)
   - [6.5 Updating a `Timer`](#65-updating-a-timer)
-- [7. _Advanced/Automated_ `API` Testing Using `Hoppscotch`](#7-advancedautomated-api-testing-using-hoppscotch)
-  - [7.0 `Hoppscotch` Setup](#70-hoppscotch-setup)
-  - [7.1 Using `Hoppscotch`](#71-using-hoppscotch)
-  - [7.2 Integration with `Github Actions` with `Hoppscotch CLI`](#72-integration-with-github-actions-with-hoppscotch-cli)
-    - [7.2.1 Changing the workflow `.yml` file](#721-changing-the-workflow-yml-file)
-    - [7.2.2 Changing the `priv/repo/seeds.exs` file](#722-changing-the-privreposeedsexs-file)
+- [7. Adding `API.Tag`](#7-adding-apitag)
+  - [7.1 Updating scope in `router.ex` and tests](#71-updating-scope-in-routerex-and-tests)
+  - [7.2 Implementing `API.Tag` CRUD operations](#72-implementing-apitag-crud-operations)
+    - [7.2.1 Adding tests](#721-adding-tests)
+    - [7.2.2 Adding `JSON` encoding and operations to `Tag` schema](#722-adding-json-encoding-and-operations-to-tag-schema)
+    - [7.2.3 Implementing `lib/api/tag.ex`](#723-implementing-libapitagex)
+  - [7.3 Allowing creating `item` with `tags`](#73-allowing-creating-item-with-tags)
+    - [7.3.1 Adding tests](#731-adding-tests)
+    - [7.3.2 Implementing `list_person_tags_text/0`](#732-implementing-list_person_tags_text0)
+    - [7.3.3 Updating `:create` in `lib/api/item.ex`](#733-updating-create-in-libapiitemex)
+      - [7.3.3.1 Creating `tag` validating functions](#7331-creating-tag-validating-functions)
+      - [7.3.3.2 Finishing up `lib/api/item.ex`'s `create` function](#7332-finishing-up-libapiitemexs-create-function)
+  - [7.3 Fetching items with embedded tags](#73-fetching-items-with-embedded-tags)
+    - [7.3.1 Adding tests](#731-adding-tests-1)
+    - [7.3.2 Returning `item` with `tags` on endpoint](#732-returning-item-with-tags-on-endpoint)
+    - [7.3.3 A note about `Jason` encoding](#733-a-note-about-jason-encoding)
+- [8. _Advanced/Automated_ `API` Testing Using `Hoppscotch`](#8-advancedautomated-api-testing-using-hoppscotch)
```

Comment:
@LuchoTurtle please don't feel the need to keep pushing the `Hoppscotch` section down in this doc.
Renumbering is a tedious waste of time and does not actually reflect the order it was built in.
You can simply move this to it's own doc: `/lib/api/hoppscotch.md` and link to it from `API.md`.